worker/uniter/runner/jujuc/credential-get.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package jujuc
+
+import (
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+)
+
+// CredentialGetCommand implements the leader-get command.
+type CredentialGetCommand struct {
+	cmd.CommandBase
+	ctx Context
+	out cmd.Output
+}
+
+// NewCredentialGetCommand returns a new CredentialGetCommand with the given context.
+func NewCredentialGetCommand(ctx Context) (cmd.Command, error) {
+	return &CredentialGetCommand{ctx: ctx}, nil
+}
+
+// Info is part of the cmd.Command interface.
+func (c *CredentialGetCommand) Info() *cmd.Info {
+	doc := `
+credential-get returns the cloud specification specific to the unit's model.
```

Comment:
"... the cloud specification used by the unit's model." ?
Rolls off the tongue a bit nicer...

Reply:
Sounds good.